tests/psimulate/test_runner.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+import pytest
+
+from vivarium_cluster_tools.psimulate.runner import report_initial_status
+
+
+def test_report_initial_status():
+
+    number_existing_jobs = 10
+    finished_sim_metadata = pd.DataFrame(index=range(number_existing_jobs))
+    report_initial_status(number_existing_jobs, finished_sim_metadata, 100)
```

Comment:
Are we already testing that `psimulate` calls `report_initial_status` with the correct arguments?